alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -93,74 +98,77 @@ class MonitorRunner(
     private val xContentRegistry: NamedXContentRegistry,
     private val alertIndices: AlertIndices,
     clusterService: ClusterService
-) : JobRunner {
+) : JobRunner, CoroutineScope, AbstractLifecycleComponent(settings) {
```

Comment:
Seems AbstractLifecycleComponent(settings) was [deprecated in 6.7 and completely removed in 7.0](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#_deprecated_constructor_literal_abstractlifecyclecomponent_settings_settings_literal_removed), they recommend using the parameterless constructor.